change/@ni-nimble-components-e9fe99af-bb91-40c3-80e3-f5b41e2b0ff6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I imagine it will be common for teams to want to contribute their own units so I think it would be useful to have some documentation to point them to. It doesn't have to be comprehensive but it would be good to at least be able to point them at an example implementation / checklist / overview.

Could you take a stab at curating this? Could be in a follow up PR if that's easier. I'm not too picky about where it lives as long as it's linkable but some options include:
- update the spec to ensure all this info is up to date with the current implementation and gathered in one section
- add a CONTRIBUTING doc in one of these folders and link to it from the spec and/or root CONTRIBUTING doc and/or the column docs in storybook

Reply:
I've added a section to the spec with instructions for implementing a new unit.